src/server/data/memo-dao.js

```diff
@@ -0,0 +1,39 @@
+/* The MemosDAO must be constructed with a connected database object */
+function MemosDAO(db) {
+
+    "use strict";
+
+    /* If this constructor is called without the "new" operator, "this" points
+     * to the global object. Log a warning and call it correctly. */
+    if (false === (this instanceof MemosDAO)) {
```

Comment:
Codacy has a fix for the issue: [Unnecessary parentheses around expression.](https://app.codacy.com/gh/codacy-acme/nodeproject/pullRequest?prid=6407538)
```suggestion
    if (false === this instanceof MemosDAO) {
```